mappings/net/minecraft/world/GameMode.mapping

```diff
@@ -21,4 +21,6 @@ CLASS none/ait net/minecraft/world/GameMode
 	METHOD a setAbilitites (Lnone/aaj;)V
 		ARG 0 abilities
 	METHOD b getName ()Ljava/lang/String;
+	METHOD c isAdventureOrSpectator ()Z
```

Comment:
I was thinking of leaving them, but I believe mapping them like this might be better since it'll help mapping easier. Would be nice if we could mark something as "might need a better name" (yellow) and "request changes" (orange) ..?